app/routes/account.rb

```diff
@@ -34,6 +34,30 @@ class Account < Core
         current_user.reset_key
         redirect "/account"
       end
+
+      post '/account/track-mentor/invite' do
```

Comment:
For some reason I couldn't get a new route going. I made a new route `routes/track_mentor.rb` and also added it to `routes.rb` but for some reason I just kept getting a 404. Is there something more that I should do? Is it agreeable to you to have this route in the account's routes?